--- a/ui/src/reducers/playerReducer.js
+++ b/ui/src/reducers/playerReducer.js
@@ -87,8 +87,8 @@
     singer: item.artist,
     duration: item.duration,
-    musicSrc: makeMusicSrc(trackId),
-    cover: subsonic.getCoverArtUrl(
+    musicSrc: item.musicSrc || makeMusicSrc(trackId),
+    cover: item.cover || subsonic.getCoverArtUrl(
       {
         id: trackId,
         updatedAt: item.updatedAt,
