--- a/ui/src/layout/Menu.jsx
+++ b/ui/src/layout/Menu.jsx
@@ -10,6 +10,7 @@
 import albumLists from '../album/albumLists'
 import PlaylistsSubMenu from './PlaylistsSubMenu'
 import LibrarySelector from '../common/LibrarySelector'
+import RecommendIcon from '@material-ui/icons/Queue'
 import config from '../config'
 
 const useStyles = makeStyles((theme) => ({
@@ -113,19 +114,9 @@
       })}
     >
       {open && <LibrarySelector />}
-      <SubMenu
-        handleToggle={() => handleToggle('menuAlbumList')}
-        isOpen={state.menuAlbumList}
-        sidebarIsOpen={open}
-        name="menu.albumList"
-        icon={<AlbumIcon />}
-        dense={dense}
-      >
-        {Object.keys(albumLists).map((type) =>
-          renderAlbumMenuItemLink(type, albumLists[type]),
-        )}
-      </SubMenu>
-      {resources.filter(subItems(undefined)).map(renderResourceMenuItemLink)}
+      {resources
+        .filter((resource) => ['artist', 'song'].includes(resource.name))
+        .map(renderResourceMenuItemLink)}
       {config.devSidebarPlaylists && open ? (
         <>
           <Divider />
@@ -137,8 +128,16 @@
           />
         </>
       ) : (
-        resources.filter(subItems('playlist')).map(renderResourceMenuItemLink)
+        []
       )}
+      <MenuItemLink
+        to="/recommendations"
+        activeClassName={classes.active}
+        primaryText={translate('resources.recommendations.name', { _: 'Recommendations' })}
+        leftIcon={<RecommendIcon />}
+        sidebarIsOpen={open}
+        dense={dense}
+      />
     </div>
   )
 }
