--- a/ui/src/layout/Menu.jsx
+++ b/ui/src/layout/Menu.jsx
@@ -10,6 +10,7 @@
 import albumLists from '../album/albumLists'
 import PlaylistsSubMenu from './PlaylistsSubMenu'
 import LibrarySelector from '../common/LibrarySelector'
+import RecommendIcon from '@material-ui/icons/Queue'
 import config from '../config'

 const useStyles = makeStyles((theme) => ({
@@ -139,6 +140,14 @@
       ) : (
         resources.filter(subItems('playlist')).map(renderResourceMenuItemLink)
       )}
+      <MenuItemLink
+        to="/recommendations"
+        activeClassName={classes.active}
+        primaryText={translate('resources.recommendations.name', { _: 'Recommendations' })}
+        leftIcon={<RecommendIcon />}
+        sidebarIsOpen={open}
+        dense={dense}
+      />
     </div>
   )
 }
